src/main/scala/ai/privado/languageEngine/python/processor/PythonProcessor.scala

```diff
@@ -30,6 +30,7 @@ import ai.privado.model.Language
 import java.util.Calendar
 import scala.jdk.CollectionConverters.CollectionHasAsScala
 import scala.util.{Failure, Success, Try}
+import java.nio.file.{Paths, Files}
 
 object PythonProcessor {
   private val logger = LoggerFactory.getLogger(getClass)
```

Comment:
Change the file location to use the new `cpg.bin` file location at line no 122. Following is the code snippet you need to change
```
121          import java.io.File
122          val cpgFile = new File(sourceRepoLocation)
123          println(s"\n\nBinary file size -- ${cpgFile.length()} in Bytes - ${cpgFile.length() * 0.000001} MB\n")
```